src/main/java/com/microsoft/azure/vmagent/AzureVMAgentTemplate.java

```diff
@@ -369,6 +376,7 @@ public int hashCode() {
     private transient String galleryName;
     private transient String galleryImageDefinition;
     private transient String galleryImageVersion;
+    private transient boolean galleryImageSpecialized;
```

Comment:
this is only for legacy fields I think so it could read from disk during the upgrade but then not save it back.
This shouldn't be added for new fields

Reply:
okay, removed this field.